src/main/java/org/prebid/server/hooks/execution/model/Stage.java

```diff
@@ -0,0 +1,6 @@
+package org.prebid.server.hooks.execution.model;
+
+public enum Stage {
+
+    entrypoint, auction_request, bidder_request, bidder_response, auction_response
```

Comment:
Pls align with updated stages:
- rename auction_request to raw_auction_request
- add processed_auction_request
- rename bidder_response to raw_bidder_response
- add processed_bidder_response